maven-plugins/enforcer-maven-plugin/src/it/projects/gitignore-test/postbuild.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+def actualIndex = 0
+def actualLines = new File(basedir, "build.log").readLines()
+
+def findLines(actualIndex, actualLines, fname) {
```

Comment:
Can you use `JUnitLauncher` and implement the test using junit.
See `io.helidon.build.maven.cache.ProjectsTestIT.BuildLog` ; you can move it to `common/test-utils`.